### PR TITLE
Update Dist_abw400.py

### PR DESCRIPTION
**Pull request by allisonbari**
Edited original distribution and according to own function.

### CODE REVIEW

File: distributions/Dist_abw400.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+		self.x_max = 1
+
+
+	def pdf(self, x):
+		"""This is your PDF"""
+		return np.sqrt(x)/(2/3)
+
+	def mean(self):
+		"""This is the mean of the PDF"""
+		return (2/5)
```
Comment (cranmer) [inline distributions/Dist_abw400.py:17]: I got something different. Make sure you include the  normalization (2/3) when you calculate the mean

**Comment by allisonbari** [comment]
I didn’t realize I had to include the normalization! I will correct it as soon as possible.

Thank you!

Allison Wexler

**Comment by allisonbari** [comment]
Hi Prof. Cranmer,

I fixed the issues with including my normalization in my calculations. 

Here is the link to the updated file in case you need it!
https://github.com/cranmer/intro-exp-phys-II/blob/915ad92b00e1f6f9605187560fa16c358bb50d97/distributions/Dist_abw400.py <https://github.com/cranmer/intro-exp-phys-II/blob/915ad92b00e1f6f9605187560fa16c358bb50d97/distributions/Dist_abw400.py>

Sincerely,
Allison Wexler

**Comment by cranmer** [comment]
I was confused for a bit, but I see you made a new pull request. instead of linking to the file like that it's nice to link to the pull request  like this #211

**Comment by allisonbari** [comment]
Got it. Thanks!

**Pull request closed without merging**